cli/Cargo.toml

```diff
@@ -75,6 +75,7 @@ pin-project = "1.0.7"
 regex = "1.4.3"
 rand = { version = "0.8.4", features = [ "small_rng" ] }
 ring = "0.16.20"
+rustls-native-certs = "0.5.0"
```

Comment:
Instead of adding this dependency to `cli/Cargo.toml` is should be reexported in one of the extension crates (probably `deno_net`)

Reply:
Agreed, I removed some of these in other places, this one was missed.